src/encoded/types/treatment.py

```diff
@@ -91,8 +94,24 @@ def display_title(self, request, treatment_type=None, chemical=None,
                 [get_item_or_none(request, construct).get('name') for construct in constructs]
             )
             disp_title = "Transient transfection of " + plasmids + conditions
-        elif treatment_type == "Biological" and biological_agent:
-            disp_title = biological_agent + " treatment" + conditions
+        elif treatment_type == "Biological":
+            disp_title = ''
+            if biological_agent or constructs:
+                if biological_agent:
+                    disp_title = biological_agent + " treatment" + conditions
+                else:
+                    const_str = ''
+                    suffix = ''
+                    if len(constructs) > 3:
+                        suffix = ' and {} more'.format(len(constructs) - 3)
+                        constructs = constructs[0:3]
+                    const_str = ", ".join(
+                        [get_item_or_none(request, construct).get('display_title') for construct in constructs]
+                    )
+                    const_str = const_str.rstrip(", ")
+                    disp_title = const_str + suffix + " treatment" + conditions
```

Comment:
Maybe "treatment(s)" or a conditional to determine if it's one or more?

Reply:
My argument here is that even if there are more than one construct used in the treatment it is still a single treatment.  If it's relevant that multiple separate treatment regimens were employed then multiple Treatment items should be entered in the 'treatments' field and then multiple display titles, each with its specific set of agents and conditions will be listed.